src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (valueResult) {
+      return function (value) {
+        return valueResult + value;
+      };
+    },
+
+    subtract: function (valueResult) {
+      return function (value) {
+        return valueResult - value;
+      };
+    },
+
+    multiply: function (valueResult) {
+      return function (value) {
+        return valueResult * value;
+      };
+    },
+
+    divide: function (valueResult) {
+      return function (value) {
+        if (value > 0) {
+          return valueResult / value;
+        } else {
+          return 'Error';
+        }
+      };
+    },
+
+    operate: function (callback, value) {
+      this.result = callback(this.result)(value);
```

Comment:
Use the arithmetic assignment operator `+=` instead of `= callback(this.result)(value)`. However, since the callback is not returning a function that takes `this.result` as an argument, this code will not work as intended. The callback functions should not be returning functions; they should directly operate on `this.result`.